src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -1559,6 +1567,11 @@ private UpdateOption getSynchronizationCooldown() {
         return Objects.requireNonNullElse(this.customSynchronizationCooldown, SYNCHRONIZATION_COOLDOWN);
     }
 
+    @Override
+    public @NotNull <T> Acquirable<T> getAcquiredElement() {
```

Comment:
If we're overriding this anyway, couldn't `LockedElement` be generic so it returns `Aquireable<Entity>`?

Reply:
But then how could you acquire a `Player`?
I dislike the current safe as it is pretty unsafe, but I didnt find any alternative